SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialDebugger.cpp

```diff
@@ -273,35 +273,30 @@ void ASpatialDebugger::OnEntityRemoved(const Worker_EntityId EntityId)
 
 void ASpatialDebugger::ActorAuthorityChanged(const Worker_AuthorityChangeOp& AuthOp) const
```

Comment:
just changed this to check for non-auth or non auth intent ops (receiver has a stricter conditional so this should happen), also reduces indentation in method

Reply:
ah